packages/odyssey-babel-plugin/src/normalizeOpts.ts

```diff
@@ -24,10 +24,13 @@ interface NormalizedOpts extends Opts {
 export function normalizeOpts(
   babelOpts: Babel.PluginPass["opts"]
 ): NormalizedOpts {
-  const opts = babelOpts || Object.create(null);
+  const opts = babelOpts as Opts;
   const normalized = {
     include: [/\.module\.(?:scss|css)$/i],
-    identityObjectProxy: false,
+    identityObjectProxy:
+      typeof opts.identityObjectProxy === "boolean"
+        ? opts.identityObjectProxy
+        : false,
```

Comment:
This logic was moved up and reconfigured to properly explain the intent. None of this is new code, and the only reason I did it this way was to fix TS errors.